README.md

```diff
@@ -60,4 +68,49 @@ Get in touch today! [Book a demo](https://xygeni.io/book-a-demo?utm_source=jenki
     * Set the `Xygeni API Url` (https://api.xygeni.io).
     * Optionally click the `TestToken` button to check url connection and token validity.
     * Click the `Save` button. 
-    ![Xygeni Configuration](docs/images/xygeni-config.png)
\ No newline at end of file
+    ![Xygeni Configuration](docs/images/xygeni-config.png)
+
+
+
+# Xygeni Sensor Unusual Activity
+
+By installing the Xygeni sensor, user activities and actions will be monitored and suspicious events will be sent to the Xygeni server for analysis.
+
+
+# XygeniSalt Step
+
+## Use Xygeni Salt Step to generate Attestation Provenance
+
```

Comment:
I would add a  context paragraph about software attestations, provenance, and why it is important.

Software attestations provide context (metadata) about artifacts like versions, origins (provenance) of the source code and its git repository plus branch / tag, the build process from which it was created, dependencies or security checks passed. The attestation is typically a signed document that gives software consumers a trusted context on the built artifacts.  A common attestatation format is SLSA provenance.